### ISSUE TITLE
Internal networking option?

### ISSUE THREAD

**Issue by j0bb13** [Nyr/openvpn-install#72]
Hey,

First, awesome script :) Used it several times and it saved me a lot of manual commands.
I saw this line in the BIG commit:
- Removed internal networking option: this is a road warrior installer after all.

I think some people will miss this feature, since not everybody using this script is a road warrior. I personally use it because I like the way it's structured, very readable.

Usage example: We'll be having a LAN party for 40+ people soon. Some friends are abroad at the moment and will join via VPN. I don't think enabling internal networking myself will be a problem, but if it's possible to add the option again it would save me some extra clicks.

Kind regards,
Job

**Comment by Nyr** [comment]
As you say, enabling it manually is not a big burden.

I removed this option primarily because it wasn't very used (although there are sure use cases like yours), because it could complicate proper firewall support (main reason, firewall support was added on this commit) and because I evaluated the possibility of migrating to `client-to-client` instead (but decided against it too).

TL;DR: it will not be enabled at least for now. I understand it has some use cases, but can be enabled easily enough just changing two lines on the script or after install :)
